docs/changelog.rst

```diff
@@ -472,3 +472,4 @@ Changelog
 
 * :release:`1.0.0 <2014-01-22>`
 * :feature:`-` initializing ap-py
+* :feature:`-` Add "Compound" to derived types
```

Comment:
this should go in reverse order and be at the top of the file.  The `-` should correspond to your PR number, i.e. `288`.